modules/ROOT/nav.adoc

```diff
@@ -76,6 +76,9 @@ include::partial$nav-migration.adoc[]
  ** xref:reconnection-strategy-about.adoc[Reconnection Strategies]
   *** xref:reconnection-strategy-reference.adoc[Reconnection Strategy Reference]
  ** xref:sources-reference.adoc[Sources Reference]
+* xref:notifications-configuration-reference.adoc[Notifications Configuration Reference]
+* xref:setting-environment-variables.adoc[Setting Environment Variables]
```

Comment:
I think this topic: setting-environment-variables.adoc[Setting Environment Variables] should be inside "General Configurations". However, I'm not sure what conclusion we reached about the ToC.

Reply:
Makes sense to me. I will do that.

Reply:
Good idea, I will do that.